src/components/Countdown/Countdown.js

```diff
@@ -1,78 +1,17 @@
 import styled from 'styled-components/macro'
-import { ReactComponent as Play } from '../../assets/play-icon.svg'
-import { ReactComponent as Stop } from '../../assets/stop-icon.svg'
-import { useState, useEffect } from 'react'
-
-export default function Countdown({ startMinutes = 25, startSeconds = 0 }) {
-  const playIcon = <Play />
-  const stopIcon = <Stop />
-
-  const [isActive, setIsActive] = useState(false)
-  const [timerExpired, setTimerExpired] = useState(false)
-  const [[minutes, seconds], setCounter] = useState([
-    startMinutes,
-    startSeconds,
-  ])
 
+export default function Countdown({ minutes, seconds }) {
   const displayMinutes = minutes.toString().padStart(2, '0')
   const displaySeconds = seconds.toString().padStart(2, '0')
```

Comment:
is padStart an self updating method? Pls explain this for me when we have time !

Reply:
Of course. When the component is getting re-rendered each time the countdown decreases, the `padStart` method checks the length of the referenced string. If its shorter than 2 digits (as configured), a `0` is added upfront. 

Let me know if you have any further question!